docs/literate/src/files/changing_trixi.jl

```diff
@@ -0,0 +1,91 @@
+#src # Changing Trixi.jl itself
+
+# If you plan on editing Trixi.jl itself, you can download Trixi.jl locally and run it from
+# the cloned directory:
+
+
+# ## Cloning Trixi.jl
+
+
+# ### Windows
+
+# If you are using Windows, you can clone Trixi.jl by using the GitHub Desktop tool:
+# - If you do not have a GitHub account yet, create it on
+#   the [GitHub website](https://github.com/join).
+# - Download and install [GitHub Desktop](https://desktop.github.com/) and then log in into
+#   your account.
+# - Open GitHub Desktop, press `Ctrl+Shift+O`.
+# - In the opened window, paste `trixi-framework/Trixi.jl` and choose the path to the folder where
+#   you want to save Trixi.jl. Then click `Clone` and Trixi.jl will be cloned to your computer. 
+
+# Now you cloned Trixi.jl and only need to tell Julia to use the local clone as the package sources:
+# - Open a terminal using `Win+R` and `cmd`. Navigate to the folder with cloned Trixi.jl using `cd`.
+# - Create new directory `run`, enter it, and start Julia with the `--project=.` flag:
+#   ```shell
+#   mkdir run 
+#   cd run
+#   julia --project=.
+#   ```
+# - Now run the following commands to install all relevant packages:
+#   ```julia
+#   using Pkg; Pkg.develop(PackageSpec(path="..")) # Install local Trixi.jl clone
+#   Pkg.add(["OrdinaryDiffEq", "Plots"])  # Install additional packages
+#   ```
+
+# Now you already installed Trixi.jl from your local clone. Note that if you installed Trixi.jl
+# this way, you always have to start Julia with the `--project` flag set to your `run` directory,
+# e.g.,
+# ```shell
+# julia --project=.
+# ```
+# if already inside the `run` directory.
+
+
+# ### Linux
+
+# You can clone Trixi.jl to your computer executing following commands:
+# ```shell
+# git clone git@github.com:trixi-framework/Trixi.jl.git 
+# # In case of an error, try the following:
```

Comment:
"case" gets highlighted. Can this be prevented?

Reply:
I'm not sure, I try just do not use "case" here.